simplic-datastack/Simplic.DataStack/Interface/IStackTimestampService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Simplic.DataStack
+{
+    /// <summary>
+    /// Stack timestamp service
+    /// </summary>
+    public interface IStackTimestampService

```

Comment:
No Repository ?

Reply:
We have no repository set for `Simplic.DataStack`